examples/webpack-demo-vanilla-bundle/src/examples/example12.scss

```diff
@@ -0,0 +1,119 @@
+$control-height: 2.4em;
+$autocomplete-max-height: 250px;
+
+@import '@slickgrid-universal/common/dist/styles/sass/slickgrid-theme-salesforce.scss';
+@import 'bulma/bulma';
+
+$modal-header-height: 50px;
+$modal-header-bg-color: #ffffff;
+$modal-footer-height: 50px;
+$modal-footer-bg-color: #f9f9f9;
+
+.editor-modal {
+  font-family: var(--lwc-fontFamily,'Salesforce Sans', 'SalesforceSans-Regular', Arial, sans-serif);
+  z-index: 9000;
+  display: inline-block;
+  border: 1px solid #b8b8b8;
+  margin: 8px;
+  background: #ffffff;
+  box-shadow: 0 0 3px #ccc;
+  position: absolute;
+  top: 10px;
+  left: 150px;
+  width: 400px;
+  position: absolute;
+  top: 20px;
+  left: 50%;
+  transform: translate(-50%);
+
+  .btn {
+    margin-right: 5px;
+    height: 28px;
+    border-radius: 2px;
+  }
+
+  .close {
+    float: right;
+    position: absolute;
+    cursor: pointer;
+    width: 20px;
+    height: 20px;
+    margin: 1px;
+    padding: 0px;
+    font-size: 18px;
+    background-color: #ffffff;
+    border: 0px solid #9c9c9c;
+    right: 10px;
+    top: 8px;
+    color: #7d7d7d;
+    font-weight: 500;
+  }
+  .modal-title {
```

Comment:
rename to `slick-modal-title`, `slick-modal-body`, `slick-modal-header`, `slick-modal-footer`